erpnext/accounts/doctype/journal_entry/journal_entry.json

```diff
@@ -506,8 +527,36 @@
   }, 
   {
    "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "currency", 
```

Comment:
Don;t use field name "currency" for this, the field is only for specific use-case and printing, use "total_amount_currency" instead.